internal/result.go

```diff
@@ -21,10 +21,6 @@ type Result struct {
 	// Signature section from the COSE Sign1 payload.
 	Signature []byte `json:"signature,omitempty"`
 
-	// SignatureOK designates if the signature was OK (but certificate could be
-	// invalid, not trusted, expired, etc.)
-	SignatureOK bool `json:"signature_ok"`
```

Comment:
This was needed when it was possible to return a valid Result object with an invalid signature and error. The updated behavior of `nitrite.Verify()`, however, does not allow this. Thus, `SignatureOK` is not required---if you do not get an error back then the implication is that the signature is ok